src/Albany_NullSpaceUtils.cpp

```diff
@@ -351,14 +354,11 @@ setCoordinatesAndComputeNullspace(const Teuchos::RCP<Thyra_MultiVector>& coordMV
 
       ComputeNullSpace(nullSpace, coordMV, interleavedOrdering, numPDEs, computeConstantModes, physVectorDim, computeRotationModes);
 
-      TEUCHOS_TEST_FOR_EXCEPTION(
-        soln_vs.is_null(), std::logic_error,
-        "numElasticityDim > 0 and (isMueLuUsed() or isFROSchUsed()): soln_map must be provided.");
-        if (isMueLuUsed()) {
-          plist->set("Nullspace", tpetraTraitsArray);
-        } else { // This means that FROSch is used
-          plist->set("Null Space", tpetraTraitsArray);
-        }
+      if (isMueLuUsed()) {
```

Comment:
Why did you remove the Teuchos test for exception?

Reply:
I just moved it further up. If soln_vs is null, we shouldn't try to compute the nullspace.